C_samples/multiple_in_out_params_sample.c

```diff
@@ -0,0 +1,192 @@
+#include <stdio.h>
+#include <stdbool.h>
+
+#include "multiple_in_out_params_sample.h"
+
+
+/* -------------------- One in, two out params ------------------------*/
+
+void one_in_two_out_params_no_return_function(int param_in, double* param_out_one, char* param_out_two)
+{
+    printf("\r\n");
```

Comment:
Calculate some values of output parameters, i.e. `*param_out_one = param_in * 2.0`

Reply:
Same in other cases